__tests__/app.test.js

```diff
@@ -102,13 +101,59 @@ describe("GET/api/articles", () => {
         });
       });
   });
-  test("articles in the array should be sorted by date in descending order", () =>{
+  test("articles in the array should be sorted by date in descending order", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then((res) => {
+        const articles = res.body;
+        expect(articles).toBeSortedBy("created_at", { descending: true });
+      });
+  });
+});
+describe("GET/api/articles/:article_id/comments", () => {
+  test("should respond with an array of comments for given article_id, each with the correct properties", () => {
     return request(app)
-    .get("/api/articles")
-    .expect(200)
-    .then((res) => {
-      const articles = res.body;
-      expect(articles).toBeSortedBy('created_at', {descending: true})
-    })
-  })
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then((res) => {
+        expect(res.body).toHaveLength(11);
+        res.body.forEach((comment) => {
+          expect(comment).toMatchObject({
+            body: expect.any(String),
+            votes: expect.any(Number),
+            author: expect.any(String),
+            article_id: expect.any(Number),
```

Comment:
We should be specific with the article id we're getting back! We're not just checking that we get articles in general back, but specifically the ones that match the article_id passed.